src/components/MessageList/VirtualizedMessageList.tsx

```diff
@@ -179,21 +179,39 @@ const VirtualizedMessageListWithContext = <
     setNewMessagesNotification(false);
   }, [virtuoso, processedMessages, setNewMessagesNotification, processedMessages.length]);
 
+  const [newMessagesReceivedInBackground, setNewMessagesReceivedInBackground] = React.useState(
+    false,
+  );
+
+  const resetNewMessagesReceivedInBackground = useCallback(() => {
```

Comment:
@petyosi shouldn't these changes be tested?

Reply:
They should, but we don't have the infrastructure for e2e testing set up. Testing that in a simulated rendering is pointless.